docs/UserGuide.md

```diff
@@ -208,16 +231,14 @@ Expected outcome:
 
 ### Exit : `exit`
 
-Exits the program. There is no auto-save upon exit.
+Description:
+
+ * Exits GradPad. All changes made while on GradPad will be saved automatically.
```

Comment:
i think might not be necessary to mention the save part cause if im not wrong the exit command doesn't do a final save before exiting 😄

Reply:
Not too sure as well, was kinda going for the notion that saving would be done automatically throughout usage of the app, cause I realise we didnt mention anything about saving throughout, and I thought people might think of saving when they think of exiting HAHAHA

Reply:
HAHAHAHA true true true...... then either we stick with this or we must include the 'auto-save' in the other features ah